### ISSUE TITLE
This is a very serious problem and I hope it can be answered

### ISSUE THREAD

**Issue by qian-fei** [Chia-Network/chia-blockchain#5987]
Only after the plot amount reaches 2PB, the currency will not be issued. The logs are normal. Before 2PB, every day, it is strange that the larger the drawing quantity is, the greater the probability of currency issue is, but it is getting lower and lower. My version is 1.1.6

**Comment by fusionfoto** [comment]
This is not a bug. Divide your total number of plots (or total amount of storage) by the estimated network space. 

in farm summary:

take Total Size of Plots divided by Estimated Network Space.

You'll notice your ratio (or percentage) keeps going down. This is because the CHIA network is growing faster than your plots are. This is to be expected.

**Comment by qian-fei** [comment]
I am increasing every day, and my growth rate is faster than that of the whole network

**Comment by markusmazurczak** [comment]
> I am increasing every day, and my growth rate is faster than that of the whole network

Well ... That is some sort of "hard to believe" because that has to mean that because more and more people are joing the network and this means that more and more plots will be generated that you will increase your plotting speed every day by creating more and more plotting hardware

**Comment by mariano54** [comment]
Check that your lookup times are below 5 seconds (preferably below 2 seconds). Check that you are receiving signage points, passing plot filter, and have at least 20 connections.

**Comment by qian-fei** [comment]
> Check that your lookup times are below 5 seconds (preferably below 2 seconds). Check that you are receiving signage points, passing plot filter, and have at least 20 connections.

2021-05-25T12:47:28.608 harvester chia.harvester.harvester: INFO     9 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.39269 s. Total 4533 plots
2021-05-25T12:47:37.190 harvester chia.harvester.harvester: INFO     8 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.29027 s. Total 4533 plots
2021-05-25T12:47:47.825 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.42374 s. Total 4533 plots
2021-05-25T12:47:55.728 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.28487 s. Total 4533 plots
2021-05-25T12:48:04.479 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.38965 s. Total 4533 plots
2021-05-25T12:48:12.821 harvester chia.harvester.harvester: INFO     10 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.50918 s. Total 4533 plots
2021-05-25T12:48:21.268 harvester chia.harvester.harvester: INFO     9 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.28052 s. Total 4533 plots
2021-05-25T12:48:29.702 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.41422 s. Total 4533 plots
2021-05-25T12:48:38.851 harvester chia.harvester.harvester: INFO     16 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.45769 s. Total 4533 plots
2021-05-25T12:48:47.816 harvester chia.harvester.harvester: INFO     10 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.44330 s. Total 4533 plots
2021-05-25T12:48:56.198 harvester chia.harvester.harvester: INFO     8 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.37638 s. Total 4533 plots
2021-05-25T12:49:07.121 harvester chia.harvester.harvester: INFO     14 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.60756 s. Total 4533 plots
2021-05-25T12:49:17.269 harvester chia.harvester.harvester: INFO     9 plots were eligible for farming 44ef8b569f... Found 0 proofs. Time: 0.32170 s. Total 4533 plots

**Comment by qian-fei** [comment]
> Check that your lookup times are below 5 seconds (preferably below 2 seconds). Check that you are receiving signage points, passing plot filter, and have at least 20 connections.

This is the log of my farmer node. It looks normal

**Comment by siaarzh** [comment]
Please follow the [Contributing Guidelines](https://github.com/Chia-Network/chia-blockchain/blob/9e21716965f6f6250f6fe4b3449a66f20794d3d9/CONTRIBUTING.md) before posting what you believe to be actual bugs and issues. By default you are given a [bug report template](https://github.com/Chia-Network/chia-blockchain/blob/main/.github/ISSUE_TEMPLATE/bug_report.md). Please, try to stick to it, as it helps the developers replicate and solve your problem.

**Anything else belongs in the [discussion](https://github.com/Chia-Network/chia-blockchain/discussions) section.**

**Comment by PapiDot** [comment]
qian-fei's question sounds rational. In general it's difficult for an average user to distinguish a bug versus a feature. The perceived Chia reward per publication is a linearly scaled by design. If a user can somehow manage to achieve a "growth rate faster than that of the whole network", his reward should increase proportional to his outpacing rate against the whole network.

qian-fei may well identify a hidden bug that most of users were unable to detect because few could afford to acquire such amount of resource in competing against the network. On the other hand, qian-fei may well identify an unannounced feature in the Chia network throttling the big players.

Either case, qian-fei's issue seems legitimate to me. Please otherwise tell users specifically what's wrong in his/her question should the question alike indeed categorized not suitable as an "issue", in order to prevent others from making the same mistake.

**Comment by siaarzh** [comment]
> qian-fei's question sounds rational. In general it's difficult for an average user to distinguish a bug versus a feature. The perceived Chia reward per publication is linearly scaled by design. If a user can somehow manage to achieve a "growth rate faster than that of the whole network", his reward should increase proportional to his outpacing rate against the whole network.
> 
> qian-fei may well identify a hidden bug that most of users were unable to detect because few could afford to acquire such amount of resource in competing against the network. On the other hand, qian-fei may well identify an unannounced feature in the Chia network throttling the big players.
> 
> Either case, qian-fei's issue seems legitimate to me. Please otherwise tell users specifically what's wrong in his/her question should the question alike indeed categorized not suitable as an "issue", in order to prevent others from making the same mistake.

TLDR: No specs, no config, no context leads me to believe he/they are miscommunicating/lying.

1. It is highly improbable that an entity capable of generating so much "hashrate" (plots) is
not able to abide by the single most important step in software engineering troubleshooting - formulating a reproducible problem statement.
2. Being able to generate plots faster than the network growth indicates substantial capital - possibly enterprise level "client", not an "average user" - in which case, a GitHub issue containing 4 free-form sentences is counterproductive.
3. Most issues created in this repository lately don't follow the guidelines. This simply means the network is growing and so is the community, which is fine. But, by not following the template, these issues create a negative precedent - an ethos of lackluster approach to problem solving and a compilation of unsolvable, unreproducible complaints and issues. At almost 6000 issues and counting, the development team probably has a hard time distinguishing real issues from user error. It is therefore imperative that those serious enough about the project behave and respond accordingly.

**Comment by johnbiggs** [comment]
We have 4747 plots and haven’t had a win in 2 weeks. Our growth is >200/day. Maybe there is an issue? Every day we check,  our rate should be a win every 5-6 days. I know it’s not guaranteed but statistically I would expect a win by 12 days. I’ll lookup the bug filing requirements and try to provide more details when I get a chance. I just wanted to post that maybe this topic shouldn’t be quickly dismissed.

**Comment by johnbiggs** [comment]
Ohh and maybe this issue is affecting lots of users and only noticed when there is an expectation of a win rate. Some little guys could simply think they have bad luck.

**Comment by lingyuj** [comment]
Same here. I have 9887 plots and haven't had a win for 8 days. Before that everything worked well.

**Comment by jly77** [comment]
Same here. I have 9887 plots and haven't had a win for 8 days. Before that everything worked well.

**Comment by keliew** [comment]
If you are using a different plotting software or part of a pool, it'd be best to check carefully if the rewards are diverted.

**Comment by qian-fei** [comment]
All the logs are normal. My daily growth rate is bigger than that of the whole network. Unfortunately, the reward I get is not proportional to my efforts. Is it really a fight for luck?

**Comment by qian-fei** [comment]
In the discussion zone, I found that many people asked the same question, but there was no official developer to answer it, so I posted here.

**Comment by keliew** [comment]
Rewards being diverted..?

**Comment by woodman733** [comment]
- No rewards for little farmers which you promise.
- No normal pools which you promise at 17thMay. And at the end of the month (and now you told about "Q&A for  pooling at June 2").
- Typical copy-paste answers for all issues and questions. Like: 'check your logs and time of responce'
People believe at this project and spend their money to this chain but no have profits and answers

**Comment by markusmazurczak** [comment]
> * No rewards for little farmers which you promise.
> * No normal pools which you promise at 17thMay. And at the end of the month (and now you told about "Q&A for  pooling at June 2").
> * Typical copy-paste answers for all issues and questions. Like: 'check your logs and time of responce'
>   People believe at this project and spend their money to this chain but no have profits and answers

I'm not trying to defend anyone, but please remember that this is a project that went "live" just under a month ago. There are now more than 1.1K issues that have been opened.
Everyone is complaining about this and complaining about that. Issues are not a place for help requests or policy discussions. How are you supposed to work on 1.1K issues as an open source DEV team, while still keeping every timeline and evolving.
I'm going for a little more understanding here.

**Comment by woodman733** [comment]
> I'm not trying to defend anyone, but please remember that this is a project that went "live" just under a month ago. There are now more than 1.1K issues that have been opened.
> Everyone is complaining about this and complaining about that. Issues are not a place for help requests or policy discussions. How are you supposed to work on 1.1K issues as an open source DEV team, while still keeping every timeline and evolving.
> I'm going for a little more understanding here.
I understand this. And sorry for my little rage quit. Also i understand that this space for technical discuss. but i don't know another place to get attention of the development team to the problems of participants

**Comment by SiberiaR** [comment]
2000 / 12 000 000 * 4608 = 0.76
The author has a 75% chance of falling out every day or a reverse 25% chance of not falling out.
By the multiplication theorem for the probability of independent events we get the following.
Probability of no coin for 5 days.
0.25*0.25*0.25*.0.25*0.25 = 0.00097
Or if the percentage is 0.00097*100% = 0.097 = 0.1
1/1000 that the block will not fall out within five days)

**Comment by johnbiggs** [comment]
> > I'm not trying to defend anyone, but please remember that this is a project that went "live" just under a month ago. There are now more than 1.1K issues that have been opened.
> > Everyone is complaining about this and complaining about that. Issues are not a place for help requests or policy discussions. How are you supposed to work on 1.1K issues as an open source DEV team, while still keeping every timeline and evolving.
> > I'm going for a little more understanding here.
> 
> I understand this. And sorry for my little rage quit. Also i understand that this space for technical discuss. but i don't know another place to get attention of the development team to the problems of participants

Well definitely not someone else’s issue to bring up different topics.  Make a new issue.

**Comment by johnbiggs** [comment]
> [2000 / 12 000 000](tel:2000 / 12 000 000) * 4608 = 0.76
> The author has a 75% chance of falling out every day or a reverse 25% chance of not falling out.
> By the multiplication theorem for the probability of independent events we get the following.
> Probability of no coin for 5 days.
> 0.25_0.25_0.25*.0.25_0.25 = 0.00097 Or if the percentage is 0.00097_100% = 0.097 = 0.1
> 1/1000 that the block will not fall out within five days)

I have 4950 plots and no wins for 14 days. While my plots were smaller 14 days ago, the ratio to netspace has remained consistent.  It should be very unlikely that I don’t have a win. 

Machine: dedicated intel nuc 10 i7 for a farmer/full node and gui. Ubuntu 20.04. No 3rd party chia software. Logging set to at least info or debug on all machines. 
5 harvesters. All report in and their plots show in the Last Attempted Proof section. 

There are no issues in my logs on the farmer or harvester. Normal x plots eligible for farming, 0 proofs found. Signage points look in order. Times are all below 1 second. 

Every machine is checked twice daily. Sampling of plots are regularly run through the checker on the farmer with 0 issues.

**Comment by keliew** [comment]
Assuming that block height is always at synced...

How many peers are connected _and_ are they also sync-ed to highest block?

**Comment by johnbiggs** [comment]
I’ve never been out of sync except for 3 times that I restarted. And the Sync only took 10 seconds. 

My block height matches the consensus and is way ahead of chia explorer. I know thats not a measurement but some indicator. 

I have ~60 peers. 51 of them are in consensus and stay with my block height. I used this command:
chia show -c | grep 340011 | wc -l

i realize thats just the moment in time but 51 agreed at that exact moment.

**Comment by keliew** [comment]
Hmm, that seems fine too.

Since you're on the GUI, I'm curious...what if you run headless, i.e. without GUI. Would the results be better...? Am unsure if it's do-able, since I don't run on GUI.

**Comment by johnbiggs** [comment]
I will try headless but it could take days to prove/not prove.

**Comment by johnbiggs** [comment]
I've been writing a script. It does reporting and some auto checks... I still see no issues. This script does check that harvesters did respond to the last query but it does not show harvester plot counts yet. Chia doesn't log harvester plot counts or allow access although they get it somehow in the GUI. Total plots >5200

Output:
Disk Free
/dev/sda2       457G   19G  416G   5% /
/dev/nvme0n1p1  1.9T  724G  1.1T  41% /media/temp
/dev/sdb1        13T   12T   19G 100% /media/plots/plots3
/dev/sdd1        11T   11T   89G 100% /media/plots/plots4
/dev/sdc1        13T   12T   19G 100% /media/plots/plots13
/dev/sdf1        13T   12T  120G 100% /media/plots/plots2
/dev/sdg1        13T   12T     0 100% /media/plots/plots11
/dev/sdk1        13T   12T  121G 100% /media/plots/plots7
/dev/sdm1        13T   12T   87G 100% /media/plots/plots6
/dev/sdn1        13T   12T  2.9G 100% /media/plots/plots5
/dev/sde1        13T  7.5T  4.6T  62% /media/plots/plots12
/dev/sdj1        13T   12T   43G 100% /media/plots/plots10
/dev/sdi1        13T   12T  120G 100% /media/plots/plots8
/dev/sdl1        13T   12T     0 100% /media/plots/plots9
/dev/sdh1        15T   14T   51G 100% /media/plots/plots1

****  PLOTTING  ****
Plotman Not Running on this machine.
Possible running plots - 5
Delays

**** Chia Status ****
Generating Temp File...
Daemons - 1   Nodes - 12   Farmers - 1   Wallets - 1   Harvesters - 1 (6)   Time Lords - 0   Introducers - 0   Simulators - 0
Node Status - Full Node Synced   Farm Status - Farming   Wallet Status - Synced   Logging - DEBUG
Coins Farmed - 6.0   Wallet Balance - 6.0     Wallet Addresses Match
Number of connections - 60   Number in Sync - 26

Errors -  0  Last - 
Warnings - 0  Last - 
Response Times - 0.18348 0.80162 0.17866   Slow Count - 0
Last response sent - 2021-05-28T15:32:35.488
Total Plots (logged) - 1518
Plots in paths - 1531
/media/plots/plots1  - 138
/media/plots/plots2  - 120
/media/plots/plots3  - 121
/media/plots/plots4  - 103
/media/plots/plots5  - 124
/media/plots/plots6  - 121
/media/plots/plots7  - 120
/media/plots/plots8  - 120
/media/plots/plots9  - 124
/media/plots/plots10  - 124
/media/plots/plots11  - 122
/media/plots/plots12  - 73
/media/plots/plots13  - 121

**Comment by keliew** [comment]
And how often do your plots get through the [filter](https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-plot-filter-and-why-didnt-my-plot-pass-it)?

**Comment by johnbiggs** [comment]
On some harvesters, at least one in every check. I just inspected my farmer and several harvesters. Plots are marked eligible regularly and checked for proof.
